### PR TITLE
[icons] feat: new icon "axle"

### PR DESCRIPTION
**Pull request by tanmaybansal0**
#### Checklist

- [ ] Includes tests
- [ ] Update documentation

#### Changes proposed in this pull request:

Added axle icon

#### Reviewers should focus on:

Making sure everything renders okay and the right files were changed

#### Screenshot

axle
axle

### CODE REVIEW

**Comment by adidahiya** [comment]
<h3>feat: new icon - axle</h3>
Build artifact links for this commit: <strong><a href="https://output.circle-artifacts.com/output/job/f61e7537-c13a-43a0-b3e3-321a7b041184/artifacts/0/packages/docs-app/dist/index.html">documentation</a> | <a href="https://output.circle-artifacts.com/output/job/f61e7537-c13a-43a0-b3e3-321a7b041184/artifacts/0/packages/landing-app/dist/index.html">landing</a> | <a href="https://output.circle-artifacts.com/output/job/f61e7537-c13a-43a0-b3e3-321a7b041184/artifacts/0/packages/table-dev-app/dist/index.html">table</a> | <a href="https://output.circle-artifacts.com/output/job/f61e7537-c13a-43a0-b3e3-321a7b041184/artifacts/0/packages/demo-app/dist/index.html">demo</a></strong>

<em>This is an automated comment from the deploy-preview CircleCI job.</em>

File: packages/icons/icons.json [hunk 1]
```diff
@@ -3967,5 +3967,12 @@
         "tags": "intelligence, intel, observe, watch, capture, humint",
         "group": "miscellaneous",
         "codepoint": 62263
+    },
+    {
+    "displayName": "Axle",
+    "iconName": "axle",
+    "tags": "axle, shaft, rail, train, wheels",
+    "group": "interface",
+    "codepoint": 62264
```
Comment (adidahiya) [inline packages/icons/icons.json]:
```suggestion
        "displayName": "Axle",
        "iconName": "axle",
        "tags": "axle, shaft, rail, train, wheels",
        "group": "interface",
        "codepoint": 62264
```

**Review by adidahiya — CHANGES_REQUESTED** [review]
failing formatting checks because of bad indentation

File: packages/icons/icons.json [hunk 2]
```diff
@@ -3967,5 +3967,12 @@
         "tags": "intelligence, intel, observe, watch, capture, humint",
         "group": "miscellaneous",
         "codepoint": 62263
+    },
+    {
+    "displayName": "Axle",
+    "iconName": "axle",
+    "tags": "axle, shaft, rail, train, wheels",
+    "group": "interface",
```
Comment (adidahiya) [inline packages/icons/icons.json]: also I don't think this is an interface icon

```suggestion
    "group": "miscellaneous",
```

**Review by adidahiya — CHANGES_REQUESTED** [review]

File: packages/icons/icons.json (hunk 1 above)
Comment (tanmaybansal0) [reply packages/icons/icons.json]: Updated

**Comment by adidahiya** [comment]
<h3>update formatting</h3>
Build artifact links for this commit: <strong><a href="https://output.circle-artifacts.com/output/job/03c67c55-f443-4284-ba9a-c670546d2a00/artifacts/0/packages/docs-app/dist/index.html">documentation</a> | <a href="https://output.circle-artifacts.com/output/job/03c67c55-f443-4284-ba9a-c670546d2a00/artifacts/0/packages/landing-app/dist/index.html">landing</a> | <a href="https://output.circle-artifacts.com/output/job/03c67c55-f443-4284-ba9a-c670546d2a00/artifacts/0/packages/table-dev-app/dist/index.html">table</a> | <a href="https://output.circle-artifacts.com/output/job/03c67c55-f443-4284-ba9a-c670546d2a00/artifacts/0/packages/demo-app/dist/index.html">demo</a></strong>

<em>This is an automated comment from the deploy-preview CircleCI job.</em>

**Review by adidahiya — APPROVED** [review]
looks fine in PR preview:

image

**Pull request merged**